cdm/src/main/java/ucar/nc2/ft/SimpleGeometryStandardFactory.java

```diff
@@ -36,7 +36,7 @@ private boolean match(FeatureType wantFeatureType, FeatureType covType) {
 
   public FeatureDataset open(FeatureType ftype, NetcdfDataset ncd, Object analysis, CancelTask task, Formatter errlog) throws IOException {
 
-	  return new ucar.nc2.dt.grid.GridDataset( ncd);
+	  return new ucar.nc2.ft2.coverage.simpgeometry.SimpleGeometryFeatureDataset( ncd);
```

Comment:
Usually the use of fully-qualified package names is reserved for situations where two classes have the same name, for example java.util.List and java.awt.List. In this case, it does not appear that there is a competing class with the same name. To improve readability, could this type be declared in the imports instead of putting the fully-qualified name inline?

I say "usually" because this isn't a cut-and-dry thing. As usual, opinions abound on [StackOverflow](https://stackoverflow.com/questions/11907245/java-import-statement-vs-fully-qualified-name). If you find the other arguments compelling, feel free to stay the course.